app/src/main/cpp/skyline/services/ldn/IUserLocalCommunicationService.h

```diff
@@ -7,16 +7,204 @@
 
 namespace skyline::service::ldn {
     namespace result {
-        static constexpr Result DeviceDisabled{203, 22};
+        constexpr Result AirplaneModeEnabled{203, 23};
+        constexpr Result InvalidInput{203, 96};
     }
 
+    constexpr size_t SsidLengthMax = 32;
+    constexpr size_t UserNameBytesMax = 32;
+    constexpr i32 NodeCountMax = 8;
+    constexpr size_t AdvertiseDataSizeMax = 384;
+    constexpr size_t PassphraseLengthMax = 64;
+
+    enum class State : u32 {
+        None,
+        Initialized,
+        AccessPointOpened,
+        AccessPointCreated,
+        StationOpened,
+        StationConnected,
+        Error,
+    };
+
+    enum class DisconnectReason : i16 {
+        Unknown = -1,
+        None,
+        User,
+        System,
+        DestroyedByUser,
+        DestroyedBySystemRequest,
+        Admin,
+        SignalLost,
+    };
+
+    enum class WifiChannel : i16 {
+        Default = 0,
+        Wifi24_1 = 1,
+        Wifi24_6 = 6,
+        Wifi24_11 = 11,
+        Wifi50_36 = 36,
+        Wifi50_40 = 40,
+        Wifi50_44 = 44,
+        Wifi50_48 = 48,
+    };
+
+    enum class LinkLevel : i8 {
+        Bad,
+        Low,
+        Good,
+        Excellent,
+    };
+
+    enum class PackedNetworkType : u8 {
+        None,
+        General,
+        Ldn,
+        All,
+    };
+
+    enum class SecurityMode : u16 {
+        All,
+        Retail,
+        Debug,
+    };
+
+    enum class AcceptPolicy : u8 {
+        AcceptAll,
+        RejectAll,
+        BlackList,
+        WhiteList,
+    };
+
+    enum class NodeStateChange : u8 {
+        None,
+        Connect,
+        Disconnect,
+        DisconnectAndConnect,
+    };
+
+    struct IntentId {
+        u64 localCommunicationId;
+        u8 _pad0_[0x2];
+        u16 sceneId;
+        u8 _pad1_[0x4];
+    };
+    static_assert(sizeof(IntentId) == 0x10);
+
+    struct SessionId {
+        u64 high;
+        u64 low;
+
+        bool operator==(const SessionId&) const = default;
```

Comment:
Not sure why you would need to explicitly default the equality operator here, is there a good reason for it?

Reply:
actually not needed, was just testing locally and forgot to remove